tests/shtest

```diff
@@ -293,20 +293,44 @@ if [ "$($VALGRIND $Q $JQ -n '{"a":"xyz"} | halt_error(1)' 2>&1)" != '{"a":"xyz"}
 fi
 
 # Check $JQ_COLORS
+## Default colors, null input
 $JQ -Ccn . > $d/color
```

Comment:
Please prepend `JQ_COLORS= ` this line and L307. `make check` fails when exporting the environment variable.